accepted/2024/improve-support-for-packages-signed-with-trusted-signing.md

```diff
@@ -0,0 +1,155 @@
+# Improve support for NuGet packages that are signed with a Trusted Signing certificate
+
+- Author Name: Dirk Lemstra [dlemstra](https://github.com/dlemstra)
+- GitHub Issue: https://github.com/NuGet/NuGetGallery/issues/10027
+
+## Summary
+
+This proposal is about improving the support for NuGet packages that are signed with a Trusted Signing certificate. This proposal
+consists of three steps to improve the experience for users that sign their packages with Trusted Signing. These three steps can
+be released separately but I can also understand that the NuGet team would want to launch this as a complete feature.
+
+## Motivation
+
+Earlier this year [Trusted Signing](https://learn.microsoft.com/en-us/azure/trusted-signing/) was launched by Microsoft and 
+recently support for this was added to `dotnet sign`. Packages signed with this can be uploaded to the NuGet Gallery. But
```

Comment:
The signing tool name isn't `dotnet sign`, it's `Sign CLI`. Would probably be useful to have it as a link to the repo.

Reply:
I have corrected the name and included a link.